main/res/layout/logcache_trackable_item.xml

```diff
@@ -73,4 +75,4 @@
         tools:text="action"
         android:maxLines="1" />
 
-</LinearLayout>
```

Comment:
It's orthogonal to the idea of the PR/commit, and IMHO it then should be handled separately.